src/ebonite/runtime/client/base.py

```diff
@@ -93,7 +93,7 @@ def __call__(self, *args, **kwargs):
 
 
 def _bootstrap_method(method: InterfaceMethodDescriptor):
-    logger.debug(f'Bootstraping server method "%s" with %s argument(s)...', method.name, len(method.args))
+    logger.debug(f'Bootstraping server method "{method.name}" with {len(method.args)} argument(s)...')
```

Comment:
this was on purpose, this way you avoid calling `str()` if you do not log because of the log level

Reply:
So noqa this line, then?

Reply:
Fixed